server/rest/src/main/java/org/infinispan/rest/resources/ContainerResource.java

```diff
@@ -578,6 +610,19 @@ public EventStream getEventStream() {
          return eventStream;
       }
 
+      @CacheEntryCreated
+      public CompletionStage<Void> onEntryCreated(CacheEntryCreatedEvent<?, EventLog> entry) {
+         if (!(entry instanceof EventImpl) || entry.isPre()) return CompletableFutures.completedNull();
+
+         final EventImpl<?, EventLog> eventImpl = (EventImpl<?, EventLog>) entry;
+         EventLog event = eventImpl.getValue();
+         if (event.getCategory() != EventLogCategory.LIFECYCLE) return CompletableFutures.completedNull();
+
+         final LifecycleEvent serializable = new LifecycleEvent(event);
+         final ServerSentEvent sse = new ServerSentEvent("lifecycle-event", serializable.toString());
```

Comment:
I think we should be sending this as `serializable.toJson()` in order to reduce the payload size.